bin/prettier.js

```diff
@@ -122,14 +98,30 @@ function getIntOption(optionName) {
   process.exit(1);
 }
 
+function getTrailingComma() {
+  let trailingComma;
+  switch(argv["trailing-comma"]) {
+    case undefined:
+    case "none":
+      return "none";
+    case "":
+    case "es5":
+      return "es5";
+    case "all":
+      return "all";
+    default:
+      throw new Error("Invalid option for --trailing-comma");
+  }
```

Comment:
```
prettier --trailing-comma --single-quote foo.js
```

oh awesome, this is the case I was interested in, that's great that it works with your solution :)